package.json

```diff
@@ -169,7 +169,7 @@
     "svg-spritemap-webpack-plugin": "^3.9.1",
     "tailwindcss": "^3.1.8",
     "ts-jest": "^29.0.1",
-    "typescript": "^4.6.2"
+    "typescript": "^4.8.3"
```

Comment:
Thought upgrading would help an issue I was seeing. It didn't. No harm in upgrading, though.